src/index.js

```diff
@@ -45,6 +54,25 @@ module.exports = function(session) {
         } catch (er) {
           return callback(er);
         }
+
+        if (!entity.createdAt || !this.expirationMs) {
+          return callback(null, result);
+        }
+
+        const createdAtMs = entity.createdAt.valueOf();
+        const expiresAtMs = createdAtMs + this.expirationMs;
+        const nowMs = new Date().valueOf();
+
+        if (expiresAtMs <= nowMs) {
+          this.destroy(sid, err => {
+            if (err) {
+              return callback(err);
+            }
+            return callback();
```

Comment:
should we explicitly return an `undefined` value? `callback(null, undefined)`, what gets returned if the session doesn't exist?

Reply:
I was following the style that was already used prior to my PR. I have a lunch appointment but let me do some research when I get back.

Reply:
The [API](https://github.com/expressjs/session#session-store-implementation) allows for either. If the session doesn't exist, it doesn't matter if we pass `null` or `undefined`, either way the middleware will handle it appropriately and generate a new session, see https://github.com/expressjs/session/blob/master/index.js#L483

Another popular implementation, [redis](https://github.com/tj/connect-redis/blob/master/lib/connect-redis.js), also uses `callback()`. If you think being explicit improves readability I'm happy to be more explicit.

Reply:
seems reasonable to me to use the same approach as `redis` 👍 no change necessary.